app/views/application_cert_review.html

```diff
@@ -242,22 +335,20 @@ <h2 class="govuk-heading-m">Interests of Justice</h2>
     </div>
 
 
-    <h1 class="govuk-heading-l">Print application</h1>
+    <!-- <h1 class="govuk-heading-l">Print application</h1> -->
```

Comment:
Any reason for commenting out rather than deleting?

Reply:
No - I have deleted it now